path.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logicalcluster
+
+import (
+	"encoding/json"
+	"path"
+	"regexp"
+	"strings"
+)
+
+const (
+
+	// ClusterHeader set to "<lcluster>" on a request is an alternative to accessing the
+	// cluster via /clusters/<lcluster>. With that the <lcluster> can be access via normal kube-like
+	// /api and /apis endpoints.
+	ClusterHeader = "X-Kubernetes-Cluster"
```

Comment:
not sure what it was used for

Reply:
maybe we got rid of it?

Reply:
@stevekuznetsov

Reply:
Remove: https://github.com/kcp-dev/kcp/pull/2448

Reply:
Still need to remove